lib/jsonapi/view.ex

```diff
@@ -51,6 +51,32 @@ defmodule JSONAPI.View do
         def relationships, do: []
       end
 
+  Fields may be omitted manually using the `hidden/1` function.
+
+      defmodule UserView do
+        use JSONAPI.View
+
+        def fields, do: [:id, :username, :email]
+
+        def type, do: "user"
+
+        def hidden(_data) do
+          [:email] # will be removed from the response
+        end
+      end
+
+  In order to use [sparse fieldsets](https://jsonapi.org/format/#fetching-sparse-fieldsets)
+  you must include the `JSONAPI.QueryParser` plug.
+
+  ### Advanced Fields Usage
+
+  Note that the overriddable `attributes/2` method can be used for more control
+  over which fields are serialized from your view. It is **strongly** recommended
```

Comment:
@doomspork @snewcomer I've added some documentation here about using `visible_fields/2`, and a bit of eager documentation about `attributes/2` being deprecated at some point. Frankly I think we ought to tighten the screws on `View` a tad, so this may be a bit presumptuous of me. I can peel back this warning if you disagree.

Reply:
Sounds good.  Not sure if it is worthwhile adding why `attributes` may be deprecated (just for posterity sake)

Reply:
Good point! I'll beef up the new note a bit.